Modules/_testcapi/long.c

```diff
@@ -81,13 +97,16 @@ test_longlong_api(PyObject* self, PyObject *args)
 #undef F_U_TO_PY
 #undef F_PY_TO_U
 
-/* Test the PyLong_AsLongAndOverflow API. General conversion to PY_LONG
-   is tested by test_long_api_inner. This test will concentrate on proper
-   handling of overflow.
-*/
+
+/*[clinic input]
+_testcapi.test_long_and_overflow
+
+Test the PyLong_AsLongAndOverflow API. General conversion to PY_LONG is tested by test_long_api_inner. This test will concentrate on proper handling of overflow.
```

Comment:
```suggestion
Test the PyLong_AsLongAndOverflow API. 

General conversion to PY_LONG is tested by test_long_api_inner. This test will concentrate on proper handling of overflow.
```

Reply:
I think that convention is to have a single empty line between the first sentence and the remaining description.